src/Ionide.ProjInfo/Library.fs

```diff
@@ -44,27 +148,32 @@ module ProjectLoader =
                 // eventSource.ErrorRaised.Add(fun t -> writer.WriteLine t.Message) //Only log errors
                 eventSource.AnyEventRaised.Add(fun t -> writer.WriteLine t.Message)
 
-            member this.Parameters : string = ""
+            member this.Parameters: string = ""
 
             member this.Parameters
                 with set (v: string): unit = printfn "v"
 
             member this.Shutdown() : unit = ()
-            member this.Verbosity : LoggerVerbosity = LoggerVerbosity.Detailed
+            member this.Verbosity: LoggerVerbosity = LoggerVerbosity.Detailed
 
             member this.Verbosity
                 with set (v: LoggerVerbosity): unit = () }
 
-    let getTfm (path: string) =
-        let pi = ProjectInstance(path)
+    let getTfm (path: string) readingProps =
```

Comment:
we need to use a `ProjectInstance` with global properties here in order to resolve/evaluate the project the same as we do during the build, so we need to pass them in